js/script.js

```diff
@@ -0,0 +1,134 @@
+          </div>
+          <div class="speaker__content--main">
+            <h2 class="speaker__content--name">${speakers[i].title}</h2>
+            <p class="speaker__content--title">${speakers[i].speaker}</p>
+            <p class="speaker__content--description">${speakers[i].description}</p>
+          </div>
+      </div>`;
+      speakerContent.innerHTML = htmlSpeakerContent;
+    }
+  } else {
+    for (let i = 0; i < speakers.length; i += 1) {
+      htmlSpeakerContent += `
+      <div class="speaker__container">
+          <div class="speaker__content--img">
+          <img src='${speakers[i].chessImg}' alt="Chess Background" class="speaker__chess">
+          <img src='${speakers[i].img}' alt="Speaker Images" class="speaker__person">
+          </div>
+          <div class="speaker__content--main">
+            <h2 class="speaker__content--name">${speakers[i].title}</h2>
+            <p class="speaker__content--title">${speakers[i].speaker}</p>
+            <p class="speaker__content--description">${speakers[i].description}</p>
+          </div>
+      </div>`;
+      speakerContent.innerHTML = htmlSpeakerContent;
+      moreBtn.style.display = 'none';
+    }
+  }
+}
+
+function resizeWindow() {
+  htmlSpeakerContent = '';
+  if (this.window.innerWidth > 768) {
+    for (let i = 0; i < speakers.length; i += 1) {
+      htmlSpeakerContent += `
+      <div class="speaker__container">
+          <div class="speaker__content--img">
+          <img src='${speakers[i].chessImg}' alt="Chess Background" class="speaker__chess">
+          <img src='${speakers[i].img}' alt="Speaker Images" class="speaker__person">
+          </div>
+          <div class="speaker__content--main">
+            <h2 class="speaker__content--name">${speakers[i].title}</h2>
+            <p class="speaker__content--title">${speakers[i].speaker}</p>
+            <p class="speaker__content--description">${speakers[i].description}</p>
+          </div>
+      </div>`;
+      speakerContent.innerHTML = htmlSpeakerContent;
+      moreBtn.style.display = 'none';
+    }
+  } else if (this.window.innerWidth < 768) {
+    htmlSpeakerContent = '';
+    for (let i = 0; i < speakers.length - 4; i += 1) {
+      htmlSpeakerContent += `
+      <div class="speaker__container">
+          <div class="speaker__content--img">
+          <img src='${speakers[i].chessImg}' alt="Chess Background" class="speaker__chess">
+          <img src='${speakers[i].img}' alt="Speaker Images" class="speaker__person">
+          </div>
+          <div class="speaker__content--main">
+            <h2 class="speaker__content--name">${speakers[i].title}</h2>
+            <p class="speaker__content--title">${speakers[i].speaker}</p>
+            <p class="speaker__content--description">${speakers[i].description}</p>
+          </div>
+      </div>`;
+      speakerContent.innerHTML = htmlSpeakerContent;
+      moreBtn.style.display = 'block';
```

Comment:
- [ ] While you did a wonderful job creating all your articles dynamically :+1: , for semantics you should use an `article` tag for each element inside the "main program" and "speakers" section